google/cloud/storage/google_cloud_cpp_storage_grpc.cmake

```diff
@@ -0,0 +1,205 @@
+               OpenSSL::SSL
+               OpenSSL::Crypto
+               ZLIB::ZLIB)
+    add_library(google-cloud-cpp::experimental-storage-grpc ALIAS
+                google_cloud_cpp_storage_grpc)
+    google_cloud_cpp_add_common_options(google_cloud_cpp_storage_grpc)
+    target_include_directories(
+        google_cloud_cpp_storage_grpc
+        PUBLIC $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}>
+               $<INSTALL_INTERFACE:include>)
+    target_compile_options(google_cloud_cpp_storage_grpc
+                           PUBLIC ${GOOGLE_CLOUD_CPP_EXCEPTIONS_FLAG})
+    target_compile_definitions(google_cloud_cpp_storage_grpc
+                               PUBLIC GOOGLE_CLOUD_CPP_STORAGE_HAVE_GRPC)
+    set_target_properties(
+        google_cloud_cpp_storage_grpc
+        PROPERTIES EXPORT_NAME "google-cloud-cpp::experimental-storage-grpc"
+                   VERSION ${PROJECT_VERSION}
+                   SOVERSION ${PROJECT_VERSION_MAJOR})
+
+    create_bazel_config(google_cloud_cpp_storage_grpc)
+endif ()
+
+google_cloud_cpp_install_headers(google_cloud_cpp_storage_grpc
+                                 include/google/cloud/storage)
+
+# Setup global variables used in the following *.in files.
+set(GOOGLE_CLOUD_CPP_PC_NAME "The GCS (Google Cloud Storage) gRPC plugin")
+set(GOOGLE_CLOUD_CPP_PC_DESCRIPTION
+    "An extension to the GCS C++ client library using gRPC for transport.")
+string(
+    CONCAT GOOGLE_CLOUD_CPP_PC_REQUIRES
+           "google_cloud_cpp_storage"
+           " google_cloud_cpp_grpc_utils"
+           " google_cloud_cpp_storage_protos"
+           " google_cloud_cpp_rpc_status_protos"
+           " google_cloud_cpp_rpc_error_details_protos"
+           " google_cloud_cpp_common"
+           " libcurl"
+           " openssl")
+string(CONCAT GOOGLE_CLOUD_CPP_PC_LIBS "-lgoogle_cloud_cpp_storage_grpc")
+
+# Create and install the pkg-config files.
+configure_file("${PROJECT_SOURCE_DIR}/google/cloud/config.pc.in"
+               "google_cloud_cpp_storage_grpc.pc" @ONLY)
+install(
+    FILES "${CMAKE_CURRENT_BINARY_DIR}/google_cloud_cpp_storage_grpc.pc"
+    DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig"
+    COMPONENT google_cloud_cpp_development)
+
+install(
+    TARGETS google_cloud_cpp_storage_grpc
+    EXPORT storage-targets
+    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+            COMPONENT google_cloud_cpp_runtime
+    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+            COMPONENT google_cloud_cpp_runtime
+            NAMELINK_SKIP
+    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+            COMPONENT google_cloud_cpp_development)
+# With CMake-3.12 and higher we could avoid this separate command (and the
+# duplication).
+install(
+    TARGETS google_cloud_cpp_storage_grpc
+    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+            COMPONENT google_cloud_cpp_development
+            NAMELINK_ONLY
+    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+            COMPONENT google_cloud_cpp_development)
```

Comment:
ordering nit: this block is moved above the `google_cloud_cpp_install_headers()` call in the other file. move if you like.

Reply:
Fixed.